h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -123,6 +122,7 @@ private Frame prepareLevelOneFrame(String levelOneKey, Model[] baseModels, Frame
       levelOneFrame.delete_and_lock(_job);
       levelOneFrame.unlock(_job);
       Log.info("Finished creating \"level one\" frame for stacking: " + levelOneFrame.toString());
+      DKV.put(levelOneFrame);
```

Comment:
You do not need explicit `DKV.put` - unlock will save the frame into DKV, or just call `frame.upade(job)` before unlock.